l10n_pt_vat/__openerp__.py

```diff
@@ -21,18 +21,18 @@
 
 {
     "name": "Portugal - IVA",
-    "version": "8.0.0.0.2",
+    "version": "8.0.0.0.3",
     "license": "AGPL-3",
     "depends": ['account', 'base_vat'],
     "author": "Sossia, Odoo Community Association (OCA)",
     "summary": "Portuguese VAT requirements extensions",
-    "website": "http://www.sossia.pt",
-    "category": "Accounting & Finance",
+    "website": "https://github.com/OCA/l10n-portugal",
+    "category": "Localisation/Europe",
```

Comment:
I'm working on a l10n_pt_partner module and checked out l10n_es_partner
which category is Localisation/Europe.  I figured maybe I should use it
also...

-- 
Pedro Castro Silva
Sossia
917213709
A Qua, 18-11-2015 às 08:12 -0800, Daniel Reis escreveu:
> In l10n_pt_vat/__openerp__.py:
> >      "license": "AGPL-3",
> >      "depends": ['account', 'base_vat'],
> >      "author": "Sossia, Odoo Community Association (OCA)",
> >      "summary": "Portuguese VAT requirements extensions",
> > -    "website": "http://www.sossia.pt",
> > -    "category": "Accounting & Finance",
> > +    "website": "https://github.com/OCA/l10n-portugal",
> > +    "category": "Localisation/Europe",
> Why did you choose this category?
> —
> Reply to this email directly or view it on GitHub.